keras_nlp/models/bert/bert_tasks.py

```diff
@@ -80,15 +79,12 @@ def __init__(
         # TODO(jbischof): create util function when ready to load backbones in
         # other task classes (e.g., load_backbone_from_string())
         if isinstance(backbone, str):
-            if backbone not in checkpoints:
+            if backbone not in backbone_presets:
                 raise ValueError(
                     "`backbone` must be one of "
-                    f"""{", ".join(checkpoints.keys())}. """
-                    f"Received: {backbone}"
+                    f"""{", ".join(backbone_presets)}. Received: {backbone}."""
```

Comment:
minor nit, you are mixing `.join(backbone_presets)}` and `.join(backbone_presets).keys()}` in this file. lets choose one.